Makefile

```diff
@@ -195,12 +195,11 @@ stop-explorer-db: ## Stops the explorer database
 	$(STOPEXPLORERDB)
 
 .PHONY: run
-run: compile-scs ## Runs all the services
```

Comment:
Why is this step removed?

Reply:
because it's not needed to run the node and it takes time